### PR TITLE
DOC: Emphasize NumPy in Ecosystem openers

### PR DESCRIPTION
**Pull request by bjnath**
#### Brief description of what is fixed or changed

Puts NumPy in the opening sentence of each Ecosystem section,
closing issue #235.

Reduces text in the Visualization section, addressing @rgommers's concern in issue #238.

Closes gh-235.

### CODE REVIEW

**Comment by bjnath** [comment]
The sentence "NumPy **transforms the data** in the burgeoning Python visualization landscape",
might read better as  "NumPy **crunches the numbers**  in the burgeoning Python visualization landscape."

**Comment by rgommers** [comment]
Looks good, less busy and more compact:

image

**Comment by rgommers** [comment]
> The sentence "NumPy **transforms the data** in the burgeoning Python visualization landscape",
> might read better as "NumPy **crunches the numbers** in the burgeoning Python visualization landscape."

Makes sense to tweak that phrase. I'm not 100% sure "crunches the numbers" captures it - there's not all that much number crunching going on in reality. Maybe _"NumPy provides the foundation for the burgeoning ..."_ ?

**Comment by stefanv** [comment]
We struggled with this one in the paper too: problem being that NumPy is not the foundation by itself, but rather one of the essential tools in the computational pipelines of many projects.

**Comment by bjnath** [comment]
We haven't used 'foundation' in the others, so it's available.

Do you like "NumPy is a key ingredient"?

**Comment by bjnath** [comment]
or "essential ingredient"

**Comment by bjnath** [comment]
"an essential component"

**Comment by rgommers** [comment]
+1 for "an essential component" as well. the other phrases are an improvement as well, but I like this one best.

File: layouts/partials/data-science.html [hunk 1]
```diff
@@ -8,27 +8,44 @@
...
+
+                    <li><b>Evaluate:</b>
+                        NumPy,
+                        <a href="https://www.tensorflow.org">TensorFlow</a>
+                    </li>
+
                     <li>
-                        <b>Presentation:</b>
+                        <b>Display:</b>
                         <a href="./index.html/#tab-visual"> Data Visualization Tools</a>
```
Comment (rgommers) [inline layouts/partials/data-science.html:43]: New issue found: this doesn't work as well as it should; it opens the correct tab, but the view jumps back to top of the page. Noting here, can deal with it later.

File: layouts/partials/data-science.html [hunk 2]
```diff
@@ -8,27 +8,44 @@
...
+
+                    </li>
+
+                    <li><b>Model:</b>
+                        <a href="https://scikit-learn.org">scikit-learn</a>,
+                        <a href="https://www.scipy.org">SciPy</a>,
+                        <a href="https://www.statsmodels.org/stable/index.html"> statsmodels</a>.
+                    </li>
+
+                    <li><b>Evaluate:</b>
```
Comment (rgommers) [inline layouts/partials/data-science.html:36]: Will rethink the contents here - having NumPy right at the end again is a little odd perhaps.

File: layouts/partials/data-science.html [hunk 3]
```diff
@@ -53,11 +70,11 @@
...
-        etc.) as well as other core disciplines of Data Science, Data Engineering and
-        Data Visualization. Tools such as <a href="https://mlflow.org">MLFlow</a> address
-        experiment hyper-parameter and result tracking needs, while
+        Effective data analytics requires deep knowledge of the data domain (e.g.,
+        retail, healthcare, marketing, finance, social media, automation, sales, travel,
+        etc.) as well as other core disciplines of data science, data engineering, and
+        data visualization. Tools such as <a href="https://mlflow.org">MLFlow</a> address
+        experiment hyperparameter and result tracking needs, while
         <a href="https://dvc.org"> DVC</a> provides data version control for data science
         and machine learning workflows.
```
Comment (rgommers) [inline layouts/partials/data-science.html:79]: Still would like to make this tab a little more compact. Perhaps in a follow-up, all these changes look good.

**Comment by rgommers** [comment]
> +1 for "an essential component" as well. the other phrases are an improvement as well, but I like this one best.

Okay went with this, pushed a commit. If there's better ideas still, we can follow up in a new PR.

I will merge this, because there's lots of useful edits here and I want to make further changes to the content of some of these tabs.

Thanks a lot @bjnath!

**Pull request merged**